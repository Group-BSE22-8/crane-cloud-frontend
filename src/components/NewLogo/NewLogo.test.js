import React from 'react';

import { shallow } from 'enzyme';

import NewLogo from './index';

describe('Test new logo', () => {
    const NewLogoComponent = shallow(<NewLogo />);
    it('checks whether required elements rendered', () => {
        const BrandLogoType = NewLogoComponent.find("div > div");
        expect(BrandLogoType.hasClass("BrandLogoType")).toEqual(true);
        expect(BrandLogoType.text()).toBe("Crane Cloud");
    });
    it('checks rendered component matches snapshot', () => {
        expect(NewLogoComponent).toMatchSnapshot();
    });
})