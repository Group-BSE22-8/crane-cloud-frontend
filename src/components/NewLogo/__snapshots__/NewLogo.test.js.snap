// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test new logo checks rendered component matches snapshot 1`] = `ShallowWrapper {}`;
